platform_versioned_docs/version-24.1.0/enterprise/data-studios.mdx

```diff
@@ -1,84 +1,198 @@
-```
-*.tower.io -> 9090
-tower.io -> 8000
-```
+Data Studios uses the following set of domains and subdomains:
 
-Additionally, a valid wildcard ssl certificate to cover the third level domain is required.
+- The domain that you set for `TOWER_SERVER_URL`, such as `example.com`.
+- A wildcard subdomain that you must configure specifically for Data Studios. This wildcard subdomain is the parent for each unique data studios session URL, such as `abcd.example.com`.
+- The connection proxy, defined by `CONNECT_PROXY_URL`. This URL is used internally and by convention is `connect.example.com`. This domain must be a subdomain of your Platform, so you cannot use `connect.example.net` if your Platform domain is `example.com`.
 
 :::note
-We recommend using only a third-level domain if possible, as costs may increase with the number of subdomains.
-:::
\ No newline at end of file
+We recommend using only a third-level domain to minimize additional costs that you might incur from additional nesting of subdomains.
+:::
+
+## Docker Compose
+
+This guide assumes that all services will be run in the same container as the rest of your Seqera Platform services.
+
+### Prerequisites
+
+- Allow inbound traffic to port 9090 on EC2 instance
+- Allow traffic on port 9090 through AWS LB
+- An AWS Route53 wildcard DNS record, such as `*.<seqera_platform_domain>`
+
+### Procedure
+
+1. Download the Data Studios [environment configuration file](./_templates/docker/data-studios.env).
+
+1. Create an initial access token:
+
+    ```
+    client_token=$(openssl rand -base64 32 | tr -d /=+ | cut -c -32)
+    ```
+
+1. Generate an RSA public/private key pair. A key size of at least 2048 bits is recommended. In the following example, the `openssl` command is used to generate the key pair:
+
+    ```shell
+    openssl genrsa -out private.pem 2048
+    openssl rsa -pubout -in private.pem -out public.pem
+    ```
+
+1. Open the `docker-compose.yml`, and uncomment the volume mount for the PEM key file for the `backend` service in the `volumes` list. Your PEM file must be named `data-studios-rsa.pem`.
+
+    ```yaml
+    volumes:
+      - $PWD/tower.yml:/tower.yml
+      # Data studios RSA key is required for the data studios functionality. Uncomment the line below to mount the key.
+      #- $PWD/data-studios-rsa.pem:/data-studios-rsa.pem
+    ```
+
+1. Open `data-studios.env` in an editor, and make the following changes:
+   1. Uncomment the `connect-proxy` and `connect-server` services.
+   1. Set the following environment variables:
+      - `PLATFORM_URL`: Specify the same value assigned to `TOWER_SERVER_URL`.
+      - `CONNECT_PROXY_URL`: Specify a base domain name for the connect proxy subdomain. For example, if you specify `example.com` then `connect.example.com` is the fully qualified domain name for the proxy.
+      - `CONNECT_OIDC_CLIENT_REGISTRATION_TOKEN`: Specify the base64-encoded value that you captured in the `client_token` environment variable.
```

Comment:
@llewellyn-sl, I wasn't sure how best to handle this -- do we even need a variable? To answer self, no. I used lowercase because someone might mistake it for a relevant variable, there are so many here. Or maybe I can just say paste the value from the terminal that you got from the previous step? I just want someone to put that value in here in some intuitive way.